src/consolelogger.c

```diff
@@ -221,10 +234,23 @@ void consolelogger_log(LOG_CATEGORY log_category, const char* file, const char*
     (void)vprintf(format, args);
     va_end(args);
 
+#if (defined(_MSC_VER) && defined(_DEBUG))
```

Comment:
>_MSC_VER [](start = 13, length = 8)

If someone builds this in non-Visual Studio compiler for Windows (and I know customers do this on our SDK), I think they'd still want OutputDebugString().  Could we change the _MSC_VER to be an #ifdef WINDOWS instead?

Reply:
A very interesting point. Yeah. mingw for the win :-). This should be done throughout the codebase seems like.

---
In reply to: [332286415](https://github.com/Azure/azure-c-shared-utility/pull/394#discussion_r332286415) [](ancestors = 332286415)

Reply:
As a separate task of course :-)

---
In reply to: [332300304](https://github.com/Azure/azure-c-shared-utility/pull/394#discussion_r332300304) [](ancestors = 332300304,332286415)